### PR TITLE
Improve support for old NV14 Radio Module FW

### LINKED ISSUE #1097: NV14 RF Signal (telemetry)
have searched but not found.

When i bind my Fli14+ rx to my NV14 (current nightly build), the NV14 repeatedly "rf signal critical".
Also the db measurement shows 20db lower then the last openTX build by Stefan and Jacob (version from fpv-racing-forum).

in openTX it shows about 73db and edgeTX whows only 53db at the same distances.

### PR DESCRIPTION
**Pull request by eshifri**
Fixes #1097 

Summary of changes: if the RM FW version is old and the Sgnl value is low [0, 10] use scaled RSSI (OTX formula) for alarm.

### CODE REVIEW

File: radio/src/telemetry/flysky_nv14.cpp [hunk 1]
```diff
@@ -125,30 +125,41 @@ int32_t GetSensorValueFlySkyNv14(const FlyskyNv14Sensor* sensor,
   int32_t value = 0;
   const nv14SensorData* sensorData =
       reinterpret_cast<const nv14SensorData*>(data + sensor->offset);
+  static bool oldRmFm = false;
```
Comment (pfeerick) [inline radio/src/telemetry/flysky_nv14.cpp:128]: I would prefer if this was less vague... older than what version? ;) `fwPreVXYZ` or something along  those lines maybe?

File: radio/src/telemetry/flysky_nv14.cpp [hunk 2]
```diff
@@ -125,30 +125,41 @@ int32_t GetSensorValueFlySkyNv14(const FlyskyNv14Sensor* sensor,
...
+
   if (sensor->bytes == 1)
     value = sensorData->UINT8;
   else if (sensor->bytes == 2)
     value = sensor->issigned ? sensorData->INT16 : sensorData->UINT16;
   else if (sensor->bytes == 4)
     value = sensorData->UINT32;
 
+  if (NV14internalModuleFwVersion <= 0x1000E) {
+    if (sensor->id == FLYSKY_SENSOR_RX_SIGNAL) {
```
Comment (pfeerick) [inline radio/src/telemetry/flysky_nv14.cpp:138]: Is the two stage check is needed because there are two firmwares that report the same version number?

Comment (eshifri) [reply radio/src/telemetry/flysky_nv14.cpp:138]: Yes, exactly: there are two FWs with the same version and different behavior.

File: radio/src/telemetry/flysky_nv14.cpp (hunk 1 above)
Comment (eshifri) [reply radio/src/telemetry/flysky_nv14.cpp:128]: The oldest ever available. :-)
I will change it.

**Pull request merged**